regexes/device/mobiles.yml

```diff
@@ -18344,7 +18394,7 @@ X-TIGI:
 
 # Xiaolajiao (Shenzhen Peppers)
```

Comment:
set description: `Xiaolajiao or Shenzhen Peppers (xiaolajiao.com)`  (this is optional)